packages/web-components/fast-foundation/src/design-system-provider/design-system-provider.ts

```diff
@@ -322,15 +323,11 @@ export class DesignSystemProvider extends FASTElement
     constructor() {
         super();
 
-        if (supportsAdoptedStylesheets && this.shadowRoot !== null) {
+        if (supportsAdoptedStylesheets) {
             const sheet = new CSSStyleSheet();
-            sheet.insertRule(":host{}");
-            (this.shadowRoot as any).adoptedStyleSheets = [
-                ...(this.shadowRoot as any).adoptedStyleSheets,
-                sheet,
-            ];
-
-            this.customPropertyTarget = (sheet.rules[0] as CSSStyleRule).style;
+            const index = sheet.insertRule(":host{}");
+            this.$fastController.addStyles(ElementStyles.create([sheet]));
```

Comment:
You can remove the custom style sheet creation here and just do this:

```ts
this.$fastController.addStyles(ElementStyles.create([":host{}"]));
```

ElementStyles will create the sheet for you. Not a big deal but would reduce the code a bit.

Reply:
I need to store a reference to the sheet so I can write properties, can I get that from ElementStyles?

Reply:
Ah, good point. The sheet itself isn't exposed since it's an abstraction over sheets. Probably keep the code you've got in this case 😄